doc/usage.rst

```diff
@@ -2,14 +2,14 @@ Usage
 =====
 
 
-The page organisation
+The Page Organisation
 ~~~~~~~~~~~~~~~~~~~~~
```

Comment:
It is somewhat typical to capitalize section headings like this.  While not necessarily required, this is not incorrect.